src/utilities.ts

```diff
@@ -76,6 +78,16 @@ export function getRequestInformation(
             };
         }
         case 'GET': {
+            if (urlPath.includes('$export')) {
+                let exportType: ExportType = 'system';
+                if (urlPath.includes('/Patient/')) {
+                    exportType = 'patient';
+                }
+                if (urlPath.includes('/Group/')) {
+                    exportType = 'group';
+                }
+                return { operation: 'read', exportType };
+            }

```

Comment:
I don't think that we should modify this function.

In general I think that `getRequestInformation` does not belong in the interface package. It deals strictly with routing concerns. Also, this new code is specific to export routing.

I think it'd be better to add this new code as a small function on  the new `exportRoute.ts`. 
We can keep `getRequestInformation ` as is and maybe later refactor it into the routing package.

Reply:
I agree that maybe `getRequestInformation` should be in the routing package. However, I think that the logic for parsing an export request should still live in the `getRequestInformation` method. This method is responsible for parsing the request, and I think it would be confusing to have two different method to parse a request.

Reply:
I disagree. 

I feel like this `getRequestInformation` is duplicating a bunch of logic with the router.

i.e. we have the new route  `this.router.get('/Patient/\\$export', ...)`. Just by route matching we already know that it is an export and we can just add `exportType = 'patient';` in that route. There's no need to duplicate that here. This is not DRY.

The only reason I see to keep it this way is to be consistent with the current codebase. But then again, I don't think the `getRequestInformation` function was needed in the first place.

Reply:
Nevermind. Now I realize that this function is called on the AuthZ middleware. With that it'd be costly to refactor to avoid the duplication with the routes in the router. It's ok to keep this as is.